components/cdc/src/observer.rs

```diff
@@ -252,7 +264,7 @@ mod tests {
         };
 
```

Comment:
Please also add the unit in https://github.com/tikv/tikv/pull/12110/files#diff-05bdfd7edb585436e8424bb42e74a6f92d53b4a1bcae9212d9d717746b2a5493R285